n_helpers.c

```diff
@@ -390,17 +442,23 @@ const char * NoteBinaryReset(void)
   @param  offset  The offset where the `data` buffer should be appended to the
                   unencoded binary data already residing on the Notecard. This
                   does not provide random access, but rather ensures alignment
-                  between the callers expectation and Notecard.
+                  across sequential writes.
   @returns  NULL on success, else an error string pointer.
   @note  Buffers are encoded in place, the buffer _MUST_ be larger than the data
          to be encoded. The original contents of the buffer will be modified.
-  @note  You may use `NoteBinaryRequiredTxBuffer()` to calculate the required size
+  @note  You may use `NoteBinaryRequiredBuffer()` to calculate the required size
          for the buffer pointed to by the `data` parameter.
 */
 /**************************************************************************/
 const char * NoteBinaryTransmit(uint8_t * data, size_t dataLen, size_t bufLen,
                                 size_t offset)
 {
+    // Validate parameter(s)
+    if (!dataLen) {
+        NOTE_C_LOG_ERROR("dataLen cannot be NULL");
```

Comment:
dataLen isn't a pointer.

Also, should we validate the other parameters, to be consistent?